tq/api.go

```diff
@@ -57,7 +58,7 @@ func (c *tqClient) Batch(remote string, bReq *batchRequest) (*BatchResponse, err
 	tracerx.Printf("api: batch %d files", len(bReq.Objects))
 
 	req = c.LogRequest(req, "lfs.batch")
-	res, err := c.DoWithAuth(remote, req)
+	res, err := c.DoWithAuth(remote, lfsapi.WithRetries(req, c.MaxRetries))
```

Comment:
Something is wrong here. `c.MaxRetries` seems to be always `0`.

Reply:
@ttaylorr Any idea about this? `c.MaxRetries` is still `0` for me with the latest change although I have configured `lfs.transfer.maxretries=10`. Any idea why?